src/AvaloniaEdit.TextMate/TextTransformation.cs

```diff
@@ -18,7 +18,7 @@ public class ForegroundTextTransformation : TextTransformation
         public Action<Exception> ExceptionHandler { get; set; }
         public int ForegroundColor { get; set; }
         public int BackgroundColor { get; set; }
-        public int FontStyle { get; set; }
+        public FontStyle FontStyle { get; set; }
```

Comment:
I haven't noticed before, but changing public property type is an ABI breaking change. Might not be critical, but needs to be carefully considered.

Reply:
Yes, this is an API breaking from TextMateSharp ...